### ISSUE TITLE
List of issues waiting on youtube-dl changes

### ISSUE THREAD

**Issue by section83** [section83/MacYTDL#4]
1.	Keep in touch with [YTDL commits for iView](https://github.com/ytdl-org/youtube-dl/pull/16755).
	A pull request has been lodged for changes to the iView extractor. No change as at 22/8/19.
2.	Overwrite file warning should have a “Continue” option.
	NB: This is subject to a [possible change to youtube-dl](https://github.com/ytdl-org/youtube-dl/pull/20405). No change as at 22/8/19.
	Example on 5/1/19 when there was a video named “video-being-downloaded.mp4.part”. MacYTDL popped the “Do you want to overwrite” box which only has options to overwrite, new-name or cancel. Need to make the “continue” alternative visible.
	Current behaviour:
	“Over-write existing” set to “Yes” doesn’t. ytdl by default doesn’t download again if it has already been downloaded. For example:
	“Chrysler_Wake_Run_-_Classic_Restos_-_Series_38.mkv has already been downloaded and merged”
	“--no-continue” makes no difference – ytdl still does NOT overwrite.
	ytdl doesn’t distinguish between partial and completed downloads.
	Code first gets prefs setting: “Over-writes Allowed” – Yes or No.
	If setting is false “--no-overwrites” is added to youtube-dl command.
	Means “Do not overwrite files”.
	“Overwrite” option works – it deletes.
	Could just add a “Continue” option which would put “--continue” into youtube-dl command.
	Means replacing "Display Dialog" with a DTP dialog.

**Comment by section83** [comment]
youtube-dl is not being maintained. MacYTDL can use _YT-DLP_ and these functions will be folded into development plan.
